akka-cluster-sharding-typed/src/test/java/akka/cluster/sharding/typed/javadsl/ShardingEventSourcedEntityWithEnforcedRepliesCompileOnlyTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding.typed.javadsl;
+
+import akka.actor.testkit.typed.javadsl.TestKitJunitResource;
+import akka.actor.typed.ActorRef;
+import akka.cluster.typed.Cluster;
+import akka.cluster.typed.Join;
+import akka.persistence.typed.ExpectingReply;
+import akka.persistence.typed.javadsl.*;
+import org.junit.ClassRule;
+
+public class ShardingEventSourcedEntityWithEnforcedRepliesCompileOnlyTest {
+
+  @ClassRule public static final TestKitJunitResource testKit = new TestKitJunitResource();
```

Comment:
Not needed for a compile-only test, I think this will spin up an actor system that is then never used.